### PR TITLE
Adding Cloud Volumes Tech Preview verbiage

### PR DESCRIPTION
**Pull request by datamattsson**
Too many changes to enumerate.

### CODE REVIEW

File: docs/container_storage_provider/hpe_cloud_volumes/index.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    The HPE Cloud Volumes CSP is currently in **beta** and available as a Tech Preview on Amazon EKS only. Please see the [1.5.0-beta Helm chart](https://artifacthub.io/packages/helm/hpe-storage/hpe-csi-driver/1.5.0-beta).
+
+[TOC]
+
+!!! seealso
+    There's a Tech Preview available in the [Video Gallery](../../learn/video_gallery/index.md#tech_preview_hybrid_cloud_kubernetes_using_hpe_cloud_volumes) on how to get started with the HPE Cloud Volumes CSP with the HPE CSI Driver.
+
+## Cloud requirements
+
+Always check the corresponding CSI driver version in [compatibility and support](../../csi_driver/index.md#compatibility_and_support) for basic requirements (such as supported Kubernetes version and cloud instance OS). If a certain feature is gated against any particular cloud provider it will be called out where applicable.
```
Comment (ckallianpur) [inline docs/container_storage_provider/hpe_cloud_volumes/index.md:15]: Do we need to specify which version of CSI works with CSP?

File: docs/container_storage_provider/hpe_cloud_volumes/index.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
+# Introduction
+
+The HPE Cloud Volumes CSP integrates seamlessly with the HPE Cloud Volumes Block service in the public cloud. The CSP abstracts the data management capabilities of the storage service for use by Kubernetes. The documentation found herein is mainly geared towards day two operations and reference documentation for the `StorageClass` and `VolumeSnapshotClass` parameters but also contains important HPE Cloud Volumes Block configuration details.
```
Comment (c-snell) [inline docs/container_storage_provider/hpe_cloud_volumes/index.md:3]: day two - "day-2"

File: docs/container_storage_provider/hpe_cloud_volumes/index.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+## Limitations
+
+Consult the [compatibility and support](../../csi_driver/index.md#compatibility_and_support) table for generic limitations and requirements. CSI and CSP specific limitations with HPE Cloud Volumes Block is listed below.
+
+- The Volume Group Provisioner and Volume Group Snapshotter sidecars are currently not implemented in the HPE Cloud Volumes CSP.
+- The base CSI driver parameter `description` is ignored by the CSP.
+- In some cases, your a "regionID" needs to be supplied in the `StorageClass` and in conjunction with Ephemeral Inline Volumes. Your "regionID" may only be found in the APIs. Join us on [Slack](https://slack.hpedev.io/) if you're hitting this issue (it can be seen in the CSP logs).
+
+!!! tip
+    While not a limitation in itself, iSCSI CHAP is mandatory with HPE Cloud Volumes but does not need any CSI driver configuration, the CHAP credentials are queried through the REST APIs from the HPE Cloud Volumes account session and applied automatically during runtime.
```
Comment (ckallianpur) [inline docs/container_storage_provider/hpe_cloud_volumes/index.md:53]: Is there a need to tell them how we can do it manually by adding it in values.yaml?

File: docs/container_storage_provider/hpe_cloud_volumes/index.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
+# Introduction
+
+The HPE Cloud Volumes CSP integrates seamlessly with the HPE Cloud Volumes Block service in the public cloud. The CSP abstracts the data management capabilities of the storage service for use by Kubernetes. The documentation found herein is mainly geared towards day two operations and reference documentation for the `StorageClass` and `VolumeSnapshotClass` parameters but also contains important HPE Cloud Volumes Block configuration details.
+
+!!! important 
+    The HPE Cloud Volumes CSP is currently in **beta** and available as a Tech Preview on Amazon EKS only. Please see the [1.5.0-beta Helm chart](https://artifacthub.io/packages/helm/hpe-storage/hpe-csi-driver/1.5.0-beta).
+
+[TOC]
+
+!!! seealso
```
Comment (c-snell) [inline docs/container_storage_provider/hpe_cloud_volumes/index.md:10]: See also

File: docs/container_storage_provider/hpe_cloud_volumes/index.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+### Common parameters for provisioning and cloning
+
+These parameters are mutable between a parent volume and creating a clone from a snapshot.
+
+| Parameter                       | String  | Description |
+| ------------------------------- | ------- | ----------- |
+| destroyOnDelete                 | Boolean | Indicates the backing Cloud Volume (including snapshots) should be destroyed when the PVC is deleted. Defaults to "false" which means volumes needs to be pruned manually in the Cloud Volume service. |
+| limitIops                       | Integer | The IOPS limit of the volume. The IOPS limit should be in the range 300 (default) to 20000. |
+| performancePolicy<sup>1</sup>   | Text    | The name of the performance policy to assign to the volume. Available performance policies: "Exchange", "Oracle", "SharePoint", "SQL", "Windows File Server". Defaults to "Other Workloads". |
```
Comment (ckallianpur) [inline docs/container_storage_provider/hpe_cloud_volumes/index.md:79]: It is not performancePolicy for cv-csp. Instead need to provide retention and schedule

File: docs/container_storage_provider/hpe_cloud_volumes/index.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+| schedule                        | Text    | Snapshot schedule to assign to the volumes. Available schedules: "hourly", "daily", "twicedaily", "weekly", "monthly", "none". Defaults to "daily". |
+| retentionPolicy                 | Integer | Retention policy to assign to the `schedule`. The parameter must be paired properly with the `schedule`. <br /><br /><ul><li>hourly: 6, 12, 24<li>daily: 3, 7, 14<li>twicedaily: 4, 8, 14<li>weekly: 2, 4, 8<li>monthly: 3, 6, 12</ul>Defaults to "3" paired with the "daily" `retentionPolicy`.
```
Comment (ckallianpur) [inline docs/container_storage_provider/hpe_cloud_volumes/index.md:81]: Nice

File: docs/container_storage_provider/hpe_cloud_volumes/index.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+| privateCloud<sup>1</sup>        | Text    | Override the compute instance provided VPC/VNET. |
+| existingCloudSubnet<sup>1</sup> | Text    | Override the compute instance provided subnet. |
+| automatedConnection<sup>1</sup> | Boolean | Override the HPE Cloud Volumes configured setting for connection automation. |
```
Comment (ckallianpur) [inline docs/container_storage_provider/hpe_cloud_volumes/index.md:84]: Do we need to add a note that it gets this value from cloud account if not provided?

File: docs/container_storage_provider/hpe_cloud_volumes/index.md (hunk 3 above)
Comment (c-snell) [inline docs/container_storage_provider/hpe_cloud_volumes/index.md:53]:
```suggestion
    While not a limitation, iSCSI CHAP is mandatory with HPE Cloud Volumes but does not need to be configured within the CSI driver. The CHAP credentials are queried through the REST APIs from the HPE Cloud Volumes account session and applied automatically during runtime.
```

**Review by c-snell — CHANGES_REQUESTED** [review]
Just a few tweaks.

File: docs/container_storage_provider/hpe_cloud_volumes/index.md (hunk 1 above)
Comment (datamattsson) [reply docs/container_storage_provider/hpe_cloud_volumes/index.md:15]: This is called out on top of the page. That it's a beta. The CV CSP will be listed as supported on the CSI driver page when we ship 1.5.0.

File: docs/container_storage_provider/hpe_cloud_volumes/index.md (hunk 3 above)
Comment (datamattsson) [reply docs/container_storage_provider/hpe_cloud_volumes/index.md:53]: I don't think it is. There's no sensible way for a customer to fetch their CHAP details from CV. You either need to make the API call with a HTTP client or "fake" a connection of a CV volumes to a cloud instances and copy out the parameters from cvctl. I rather not elaborate on any of these methods.

File: docs/container_storage_provider/hpe_cloud_volumes/index.md (hunk 5 above)
Comment (datamattsson) [reply docs/container_storage_provider/hpe_cloud_volumes/index.md:79]: I think you're mixing "performancePolicy" with "protectionSomething"? performancePolicy is part of the parameter list of the CSP.

File: docs/container_storage_provider/hpe_cloud_volumes/index.md (hunk 7 above)
Comment (datamattsson) [reply docs/container_storage_provider/hpe_cloud_volumes/index.md:84]: I'll add some clarifications.

File: docs/container_storage_provider/hpe_cloud_volumes/index.md (hunk 4 above)
Comment (datamattsson) [reply docs/container_storage_provider/hpe_cloud_volumes/index.md:10]: `seealso` is mapped to the blue [admonition](https://github.com/hpe-storage/scod/#admonitions).

File: docs/container_storage_provider/hpe_cloud_volumes/index.md (hunk 3 above)
Comment (datamattsson) [reply docs/container_storage_provider/hpe_cloud_volumes/index.md:53]: Thanks, I'll have this in my next commit.

File: docs/container_storage_provider/hpe_cloud_volumes/index.md (hunk 5 above)
Comment (ckallianpur) [reply docs/container_storage_provider/hpe_cloud_volumes/index.md:79]: right. I was confused with protectionTemplate

**Review by ckallianpur — APPROVED** [review]

**Review by c-snell — APPROVED** [review]
lgtm

**Pull request merged**